react/features/chat/components/web/Chat.js

```diff
@@ -96,7 +126,25 @@ class Chat extends AbstractChat<Props> {
      * @returns {ReactElement}
      */
     _renderChat() {
-        const messages = this.props._messages.map(this._renderMessage);
+        const groupedMessages = this._getMessagesGroupedBySender();
+
+        const messages = groupedMessages.map((group, index) => {
+            const messageType = group[0] && group[0].messageType;
+            let className = 'remote';
+
+            if (messageType === 'local') {
+                className = 'local';
+            } else if (messageType === 'error') {
+                className = 'error';
+            }
```

Comment:
Can we simply use the messageType as className, if they are so identical? If we ever receive a message type that we're not prepared for, it'll have an unknown class, that is essentially the same as not assigning a class at all.

Not sure what the messageType of the remote messages are though

Reply:
It's "remote". Right now all messages types match the classname, yes.